Documentation/index.rst

```diff
@@ -112,7 +136,14 @@ get started and experiment with Cilium.
 
    cheatsheet
    cmdref/index
+   helm-reference
    kvstore
    further_reading
    glossary
-   helm-reference
+
+.. toctree::
+   :maxdepth: 2
+   :caption: BPF and XDP Reference Guide
+
+   bpf
```

Comment:
This is another case where we could move the contents up a level so that the section name isn't repeated in the navigation:

<img width="312" alt="Screenshot 2022-08-23 at 17 51 48" src="https://user-images.githubusercontent.com/458616/186216843-45a649b5-52f1-4231-9bab-3c1527a755f7.png">
 
(Sorry I may have missed this one last time I reviewed!)

Reply:
I would leave the BPF guide for a follow-up though, there's a lot to do on that page. Plus it needs some extra care: if we move/rename it we want to make sure there's a redirect, this page is cited pretty much everywhere people list references for BPF...
What do you think?

Reply:
I'm fine with leaving this for a follow-up